src/core_deku/ticket_table.ml

```diff
@@ -0,0 +1,113 @@
+open Helpers
+
+module Tickets = struct
+  module Ticket_map = Map.Make_with_yojson (Ticket_id)
+
+  type t = Amount.t Ticket_map.t [@@deriving yojson]
+
+  let get_balance t ~ticket = Ticket_map.find_opt ticket t
+
+  let to_seq map = Ticket_map.to_seq map
+
```

Comment:
```suggestion
```
This is dead code that isn't used. It's also not really useful as an alias in this case.